phyui/plot/waveforms.py

```diff
@@ -325,6 +326,7 @@ class WaveformView(PanZoomCanvas):
     def __init__(self, **kwargs):
         super(WaveformView, self).__init__(**kwargs)
         self.visual = Waveforms()
+        self.resizable = Bool(defaultValue=True, sync=True)
```

Comment:
I don't think the IPython import and this should be in this file. At least it should be in PanZoomCanvas, or even better, in the IPython widget inself. PanZoomCanvas should be usable without IPython, with just VisPy+Qt for example.

Also can you do this instead:

```python
class TheWidget:
    resizable = Bool(value=True, sync=True)
```

(note that Python variables like `defaultValue` should be lowercase + underscore instead)